docs/core/deploying/index.md

```diff
@@ -1,98 +1,169 @@
 
-- Self-contained deployment. Unlike FDD, a self-contained deployment (SCD) doesn't rely on the presence of shared components on the target system. All components, including both the .NET Core libraries and the .NET Core runtime, are included with the application and are isolated from other .NET Core applications. SCDs include an executable (such as *app.exe* on Windows platforms for an application named `app`), which is a renamed version of the platform-specific .NET Core host, and a *.dll* file (such as *app.dll*), which is the actual application.
+**_Acquisition_** is about finding and getting what you need for both development (that is toolset/SDK) and for runtime. Acquisition also includes the ability to detect if the packages you have are up-to-date.
 
-- Framework-dependent executables. Produces an executable that runs on a target platform. Similar to FDDs, framework-dependent executables (FDE) are platform-specific and aren't self-contained. These deployments still rely on the presence of a shared system-wide version of .NET Core to run. Unlike an SCD, your app only contains your code and any third-party dependencies that are outside of the .NET Core libraries. FDEs produce an executable that runs on the target platform.
+**_Deployment_** is about packaging the application and delivering the application and runtime to the target machine. 
 
-## Framework-dependent deployments (FDD)
+<br/>
 
-For an FDD, you deploy only your app and third-party dependencies. Your app will use the version of .NET Core that's present on the target system. This is the default deployment model for .NET Core and ASP.NET Core apps that target .NET Core.
+## For Developers: Building and Deploying your .NET Core app
 
-### Why create a framework-dependent deployment?
+As a developer, your workflow will typically look like this:
 
-Deploying an FDD has a number of advantages:
+Step 1 | | Step 2 | | Step 3 | | Step 4 |
+|---|---|---|---|---|---|---|
+ [Build your App](#Step-1-Build-your-app) | --> | [Publish your app](#Step-2-publish-your-app) | --> | [Package your app for deployment](#Step-3-Package-your-app-for-deployment) | --> | [Deploy your app](#Step-4-Deploy-your-app) | 
 
-- You don't have to define the target operating systems that your .NET Core app will run on in advance. Because .NET Core uses a common PE file format for executables and libraries regardless of operating system, .NET Core can execute your app regardless of the underlying operating system. For more information on the PE file format, see [.NET Assembly File Format](../../standard/assembly/file-format.md).
 
-- The size of your deployment package is small. You only deploy your app and its dependencies, not .NET Core itself.
+<br/>
 
-- Unless overridden, FDDs will use the latest serviced runtime installed on the target system. This allows your application to use the latest patched version of the .NET Core runtime. 
+### Step 1: Build your app
```

Comment:
This section talks exclusively about VS and using the CLI.  I am surprised VS Code is not discussed.  Isn't this a scenarios for a lot of our users?

Reply:
Ack, added links to using .NET Core with VS Code